### ISSUE TITLE
Facebook plugin: Empty image field in Archetype News Item causes divide by zero error

### ISSUE THREAD

**Issue by rbruch** [collective/sc.social.like#31]
Using sc.social.like version 2.0b2.dev0 in a Plone 4.2.7 site.

In sc.social.like.utils.get_content_image, in this section:
```
        view, field = get_images_view(context)
        if view:
            try:
                sizes = view.getImageSize(field)
```
When the object in question is an Archetype News Item with no image uploaded, "sizes" is assigned to (0, 0). Later on in the method, "sizes" is passed to _image_size(), in which the 0 is put in the denominator of a fraction/division.

I worked around this in my site by adding:
```
            if sizes == (0, 0):
                return None
```
before line 44 of utils.py (before "if sizes:").

This prevents the divide by zero error, but I'm not sure if it has any unintended side effects.

**Comment by keul** [comment]
I think this has been closed by #34

**Comment by hvelarde** [comment]
please update [2.0](https://pypi.python.org/pypi/sc.social.like/2.0)
